pyproject.toml

```diff
@@ -34,6 +34,7 @@ dependencies = [
     "cloudpickle>=2.0.0",
     "deepmerge",
     "fs",
+    "httpx",
```

Comment:
Let's also remove requests from the deps?

Reply:
We can't in any of these PRs, it's used by everything else.

Reply:
so are you going to remove this once all of the PRs are merged?

Reply:
Yep, I think the tests PR is not working, so will probably remove it with that one.